manifests/m/MoheshwarAmarnathBiswas/YouTubeVideoDownloader/1.2.2/MoheshwarAmarnathBiswas.YouTubeVideoDownloader.installer.yaml

```
@@ -0,0 +1,17 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: MoheshwarAmarnathBiswas.YouTubeVideoDownloader
+PackageVersion: 1.2.2
+Installers:
+- Architecture: x64
+  InstallerType: inno
+  Scope: machine
+  InstallerUrl: https://github.com/fluentmoheshwar/yt-download-py/releases/download/v1.2.2/YouTube_Video_Downloader_Installer.exe
+  InstallerSha256: fc69adf7ccb0c7d25d0ec6a0ad1f13e1b3366b192fd99e58a1bf9e476967c26c
+  InstallerSwitches:
+    Custom: /ALLUSERS /VERYSILENT

```

```suggestion
    Custom: /ALLUSERS
```
Silent switches should not be part of custom